package.json

```diff
@@ -16,7 +16,8 @@
     "test": "jest --forceExit"
   },
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.3",
+    "minimist": "^1.2.6"
```

Comment:
This is not needed, because in master there is already latest version of `minimist`.

You can see that by running: `npm list minimist`
```
├─┬ @babel/eslint-parser@7.17.0
│ └─┬ @babel/core@7.16.0
│   └─┬ json5@2.1.3
│     └── minimist@1.2.6 deduped
└─┬ eslint-plugin-import@2.25.4
  └─┬ tsconfig-paths@3.14.1
    ├─┬ json5@1.0.1
    │ └── minimist@1.2.6 deduped
    └── minimist@1.2.6
```